lib/request.js

```diff
@@ -1,6 +1,6 @@
 module.exports = request
 
-const isPlainObject = require('is-plain-object')
+const isPlainObject = (x => typeof x === function ? x : x.default)(require('is-plain-object'))
```

Comment:
There must be a better solution than this :/ I hope this will be resolved by https://github.com/octokit/request.js/pull/45, but we can merge it for the time being. Can you resolve the failing tests?

Reply:
Oh wow that failing test looks complicated `node-gyp` stuff. I've thought of another way though, so I'll just try that.

Reply:
(I was just looking in the wrong place for tests, oops)